test/testcafe/spec/EnrollProfileView_spec.js

```diff
@@ -157,3 +164,25 @@ test.requestHooks(requestLogger, EnrollProfileSignUpAllBaseAttributesMock)('All
     await t.expect(await enrollProfilePage.getFormFieldLabel(selector)).eql(formFieldToLabel[formField]);
   });
 });
+
+test.requestHooks(requestLogger, EnrollProfileSignUpWithPasswordMock)('should show prompt for password and password requirements', async t => {
+  const enrollProfilePage = new EnrollProfileViewPageObject(t);
+  const identityPage = await setup(t);
+  await identityPage.clickSignUpLink();
+
+  requestLogger.clear();
+  await t.expect(enrollProfilePage.getFormTitle()).eql('Sign up');
+  await t.expect(await enrollProfilePage.getFormFieldLabel('userProfile.firstName')).eql('First name');
+  await t.expect(await enrollProfilePage.getFormFieldLabel('userProfile.lastName')).eql('Last name');
+  await t.expect(await enrollProfilePage.getFormFieldLabel('userProfile.email')).eql('Email');
+  // verify prompt & field for password are rendered
+  await t.expect(await enrollProfilePage.getFormFieldLabel('credentials.passcode')).eql('Password');
+  // verify password text toggle is rendered
+  await t.expect(await identityPage.hasShowTogglePasswordIcon()).ok();
+  // verify password requirements are rendered
+  await t.expect(await enrollProfilePage.form.elementExist('section[data-se="password-authenticator--rules"]')).ok();
+  await t.expect(await enrollProfilePage.form.getInnerTexts('div[class="password-authenticator--heading"]')).eql(['Password requirements:']);
+  await t.expect(await enrollProfilePage.form.elementExist('ul[class="password-authenticator--list"]')).ok();
+
+  await t.expect(await enrollProfilePage.getSaveButtonLabel()).eql('Sign Up');
```

Comment:
Maybe add a test to make sure the form can be submitted successfully? Also was wondering do we want a Re-enter password field to allow end user to confirm their password. Overall looks good!

Reply:
Thanks for the review. Yes, I agree about the test; I don't think the backend is fully finished yet but I have a ticket here for end to end testing when it is: https://oktainc.atlassian.net/browse/OKTA-480990

Regarding the `confirm password` field, design confirmed having 1 password field in the [Figma](https://www.figma.com/file/R2S1ukzhwQnScwT0wz3ten/Sign-up?node-id=2%3A331)